extensions/wikia/ArticleVideo/node_modules/jwplayer-fandom/src/settings.plugin.js

```diff
@@ -115,56 +112,60 @@ wikiaJWPlayerSettingsPlugin.prototype.addSettingsContent = function (div) {
 		div.appendChild(this.qualityLevelsList);
 	}
 
+	if (this.config.showCaptions) {
+		this.createCaptionsList();
+		div.appendChild(this.captionsList);
+	}
+
 	return div;
 };
 
 wikiaJWPlayerSettingsPlugin.prototype.createSettingsListElement = function () {
 	var settingsList = document.createElement('ul');
 
-	settingsList.classList.add('wikia-jw-settings__list');
-	settingsList.classList.add('wds-list');
+	settingsList.className = 'wikia-jw-settings__list wds-list';
 
 	if (this.config.showQuality) {
 		settingsList.appendChild(this.createQualityButton());
 	}
 
+	if (this.config.showCaptions) {
+		settingsList.appendChild(this.createCaptionsButton());
+	}
+
 	if (this.config.showAutoplayToggle) {
 		settingsList.appendChild(this.createAutoplayToggle());
 		this.show();
 	}
 
-	if (this.config.showCaptionsToggle) {
-		settingsList.appendChild(this.createCaptionsButton());
-		this.addCaptionListener();
-	}
-
 	return settingsList;
 };
 
-wikiaJWPlayerSettingsPlugin.prototype.createSVG = function (svgHtml) {
-	return domParser.parseFromString(svgHtml, 'image/svg+xml').documentElement;
-};
+wikiaJWPlayerSettingsPlugin.prototype.createSubmenuWrapper = function () {
+	var backElement = document.createElement('li'),
+		submenuWrapper = document.createElement('ul');
 
-wikiaJWPlayerSettingsPlugin.prototype.createQualityButton = function () {
-	var rightArrowIcon = this.createSVG(wikiaJWPlayerIcons.back);
-	var qualityButton = document.createElement('li');
+	backElement.className = 'wikia-jw-settings__back';
+	// todo change label after i18n
+	backElement.innerHTML = createArrowIcon('left').outerHTML + ' Back';
+	backElement.addEventListener('click', this.showSettingsList.bind(this));
```

Comment:
If that's not a huge PITA, I'd consider using a clickable element if we're adding click handlers (e.g. an anchor / button etc.). Not a big deal, but here are the pros:

1) Like the default JWPlayer controls, it would be 'traversable' by using tab + enter to navigate
2) It's better for accesibility, as button suggests there's something clickable here

(Point number 2 we could address by adding role=button to the clickable element).

Reply:
1. I did not create that, I've only changed `this.backButton` to `backButton`, however I can do that.
2. please  suggest changes for jwplayer code in jwplayer repo to make things consistent, here we're only copying it.